### ISSUE TITLE
Training Data Format

### ISSUE THREAD

**Issue by AhmedHessuin** [raywzy/ICT#19]
can you tell me the training data format?
as i understand 
1) training data contain images for the pretrained models 256 x 256  in directory for example **X**
2) training data contain the masks of the images same size as the original train image for my example 256x256 in directory **Y**
3) optional, if i used flag **--random_stroke**, i don't need the training data contain the masks
if i am right, please confirm and if i am wrong please tell me the right 

---
thanks in advance

**Comment by raywzy** [comment]
Sorry for the late reply due to other ddl. Basically, yes. Feel free to raise any other questions further.

**Comment by administratorssssssss** [comment]
1. Use the 1024 of the FFHQ dataset to reset to 256X256 size of the image as Y
2. The 256X256 size picture is the input X after adding MASK?
What is the format of the data, and can you send me some sample pictures? thanks！
